arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -934,7 +934,7 @@
           },
           "description": "Gets or sets the metadata attached to the resource."
         },
-        "TTL": {
+        "ttl": {
```

Comment:
I am a little unclear on what you are trying to achieve. Changing from All Caps to small letters will not change anything during generation. This will still be Pascal cased in C# for instance so `TTL` or `ttl` will be generated as `Ttl`. Could you please take a moment to explain why this change is being made?